instrumentation/jaxws/jaxws-2.0-metro-2.2-testing/build.gradle.kts

```diff
@@ -0,0 +1,28 @@
+plugins {
+  id("otel.javaagent-testing")
+}
+
+dependencies {
+  testImplementation("javax.servlet:javax.servlet-api:3.0.1")
+  testImplementation("com.sun.xml.ws:jaxws-rt:2.3.6")
```

Comment:
usually we test against the earliest supported version, I think you should be using `testLibrary("com.sun.xml.ws:jaxws-rt:2.2.0.1")` here.

Reply:
n/m [found it](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/7496a8dba76ee4f67a47c78eaf11bac83eb73ab4/conventions/src/main/kotlin/io.opentelemetry.instrumentation.base.gradle.kts#L17-L21)

Reply:
I think here it doesn't really matter which you use as there is also a `latestDepTestLibrary` that will set the version when we test with the latest version of the dependencies. Otherwise `testLibrary` dependencies would get bumped to latest version while `testImplementation` won't.